simulation_ws/src/cloudwatch_simulation/launch/world_turtlebot_navigation.launch.py

```diff
@@ -0,0 +1,99 @@
+# *******************************************************************************/
+# This launch file serves as a template for navigation launch files.
+# It takes two key argument:
```

Comment:
I think the `x_pos`, `y_pos`, and `z_pos` are key arguments as well. I think we should make these 5 key arguments non-optional, too, that is no default values for them.

Reply:
right